### PR TITLE
feat(kslideout): add persistence prop [MA-1928]

### PR DESCRIPTION
**Pull request by mihai-peteu**
# Summary
https://konghq.atlassian.net/browse/MA-1928

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: docs/components/slideout.md [hunk 1]
```diff
@@ -118,6 +118,23 @@ Tells the component whether or not to enable / disable overlay when the slideout
...
 
+### preventCloseOnBlur
+
+Persists the slideout, ignoring clicks outside of the panel. Defaults to `false`.
+
+<KToggle v-slot="{ isToggled, toggle }">
+  <div>
+    <KButton @click="toggle">Toggle persistent Panel</KButton>
+    <KSlideout :is-visible="isToggled.value" @close="toggle" :has-overlay="false" 
+    :preventCloseOnBlur="true" close-button-alignment="end" title="GET/account/kong.docs.dev">
```
Comment (kaiarrowood) [inline docs/components/slideout.md:129]:
```suggestion
    prevent-close-on-blur close-button-alignment="end" title="GET/account/kong.docs.dev">
```

Comment (kaiarrowood) [reply docs/components/slideout.md:129]: You also need to display the code snippet below

**Comment by mihai-peteu** [comment]
@kaiarrowood fixed the spec! It actually performs a legitimate check now :)

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🚀

**Review by portikM — APPROVED** [review]

**Pull request merged**